guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<p>Adds conceptual grouping of classes and sub modules in <code>mapping</code>  module instead of presenting members in pure alphabetical order</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done!</p><p> </p>